src/gui/sdl_mapper.cpp

```diff
@@ -2904,6 +2904,11 @@ void MAPPER_Run(bool pressed) {
 SDL_Surface* SDL_SetVideoMode_Wrap(int width,int height,int bpp,uint32_t flags);
 
 void MAPPER_DisplayUI() {
+	// The mapper is about to take-over SDL's surface and rendering
+	// functions, so disengage the main ones. When the mapper closes, SDL
+	// main will recreate its rendering pipeline.
+	GFX_DisengageRendering();
```

Comment:
Out of curiosity, what happened before without disengaging the renderer? Opening the mapper did pause the emulation, so did it matter that it wasn't disengaged?

Reply:
It crashes on resume when using texture-output because some of the texture pointers have been reused here. 
So this safety removes access to them until sdlmain repopulates them and re-enganges this rendering functions.

Reply:
Ooops, definitely needed then 👍🏻